state/resources_mongo.go

```diff
@@ -213,7 +213,7 @@ func newResolvePendingResourceOps(pending storedResource, exists bool) []txn.Op
 		Resource:      newRes.Resource.Resource,
 		id:            newRes.ID,
 		applicationID: newRes.ApplicationID,
-		lastPolled:    time.Now().UTC(),
+		lastPolled:    time.Now().AddDate(0, 0, 0).UTC(),
```

Comment:
please add comment here and elsewhere to explain the AddData() usage
maybe even a todo to fix it properly